cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -197,6 +222,29 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
+  @Test def use_index_with_where_using_multiple_range_comparisons_composite() {
+    executePreparationQueries(List("CREATE INDEX ON :Person(highScore, name)"))
+    // Need to make index preferable in terms of cost
+    executePreparationQueries((0 to 300).map { i =>
+      "CREATE (:Person)"
+    }.toList)
+    profileQuery(
+      title = "Multiple range comparisons using `WHERE` (composite index)",
+      text = "When the `WHERE` clause contains multiple inequality (range) comparisons for the same property, these can be combined " +
+        "in a single index range seek. " +
+        "If this range seek is for the first property in a composite index then the index may be used. " +
+        "For an index on `:Person(highScore,name)` the following query is a valid query. " +
+        "However, for an index on `:Person(name,highScore)` it is not and will instead get an exception.",
```

Comment:
I can see that this query will not use the index `:Person(name,highScore)` but will it really throw an exception?

Reply:
Currently yes, but updated the text to not mention it since it probably shouldn't.